CONTRIBUTING.md

```diff
@@ -103,8 +103,9 @@ Pull request titles must be:
 
 For example:
 
-> feat: add lodestar prover for execution api
-> refactor(reqresp)!: support byte based handlers
+- feat: add lodestar prover for execution api
```

Comment:
Not sure if we should apply some highlighting via `` or similar. Decided it is best to just have bullet points with normal text